src/com/projectkorra/projectkorra/airbending/AirSwipe.java

```diff
@@ -256,7 +256,7 @@ public void run() {
 						breakBreathbendingHold(entity);
 					} else if (entity.getEntityId() != AirSwipe.this.player.getEntityId() && !(entity instanceof LivingEntity)) {
 
-						GeneralMethods.setVelocity(entity, fDirection.multiply(AirSwipe.this.pushFactor));
+						GeneralMethods.setVelocity(AirSwipe.this,entity, fDirection.multiply(AirSwipe.this.pushFactor));

```

Comment:
Please change to `GeneralMethods.setVelocity(this, entity, fDirection.multiply(AirSwipe.this.pushFactor));`